app/screens/home-screen/home-screen.tsx

```diff
@@ -341,16 +255,18 @@ export const HomeScreen: React.FC = () => {
         <Button
           {...testProps("Settings Button")}
           buttonStyle={styles.topButton}
-          containerStyle={styles.separator}
+          // containerStyle={styles.separator}
           onPress={() => navigation.navigate("settings")}
           icon={<SettingsIcon />}
         />
       </View>
 
       <View style={styles.walletOverview}>
-        <WalletOverview loading={loading} setModalVisible={setModalVisible} />
+        <WalletOverview
+          loading={loading}
+          setIsStablesatModalVisible={setIsStablesatModalVisible}
+        />
       </View>
-
       <FlatList
```

Comment:
I would get rid of the flat list and just have a row of all the icon buttons. The refresh then needs to be added as some 'onScrollDown` prop on the screen.